draft-howard-rats-coserv.md

```diff
@@ -104,6 +105,37 @@ The terminology from CBOR {{-cbor}}, CDDL {{-cddl}} and COSE {{-cose}} applies;
 and {{Section G of -cddl}}. Terms and concepts are always referenced as proper nouns, i.e., with Capital Letters.
 
+# Aggregation and Trust Models {#secaggregation}
+
+The roles of Endorser or Reference Value Provider might sometimes be fulfilled by aggregators, which collect from multiple supply chain sources, or even from other aggregators, in order to project a holistic view of the endorsed system.
+The notion of such an aggregator is not explicit in the RATS architecture.
+In practice, however, supply chains are complex and multi-layered.
+Supply chain sources can include silicon manufacturers, device manufacturers, firmware houses, system integrators, service providers and more.
+In practical terms, an Attester is likely to be a composite entity, formed of components from across such supply chains.
+Evidence would be likewise structured, with contributions from different segments of the Attester's overall anatomy.
+A Verifier for such Evidence may find it convenient to contact an aggregator as a single source of truth for endorsements.
+An aggregator would have intelligence about the Attester's complete anatomy and supply chain.
+It would have the ability to contact all contributing supply chain actors for their individual Endorsements and Reference Values, before collecting them into a cohesive set, and delivering them to the Verifier as a single, ergonomic package.
+In pure RATS terms, an aggregator is still an Endorser or a Reference Value Provider - or, more likely, both.
+It is not a distinct role, and so there is no distinctly-modeled conveyance between an aggregator and a Verifier.
+However, when consuming from an aggregator, the Verifier may need visibility of the aggregation process, possibly to the extent of needing to audit the results by inspecting the individual inputs that came from the original supply chain actors.
+CoSERV addresses this need, and caters equally for both aggregating and non-aggregating supply chain sources.
+
+To support deployments with aggregators, CoSERV allows for flexible trust models as follows.
+
+- **Shallow Trust**: in this model, the consumer trusts the aggregator, solely and completely, to provide authentic descriptions of the endorsed system.
+The consumer does not need to audit the results of the aggregation process.
+- **Deep Trust**: in this model, the consumer has a trust relationship with the aggregator, but does not deem this to be sufficient.
+The consumer can still use the collected results from the aggregation process, where it is convenient to do so, but also needs to audit those results.
+
+Any given CoSERV transaction can operate according to either model.
+The consumer decides which model to use when it forms a query.
+The CoSERV result payload can convey both the aggregated result and the audit trail as needed.
+The payload size is inevitably smaller when the shallow model is used, but the choice between the two models is a question for implementations and deployments.
```

Comment:
While I’d expect this to be true in the vast majority of cases, there may be pathological situations where it may not hold.  I think it’s safer to remove L134.  Alternatively, we could s/inevitably/typically/.